.github/ISSUE_TEMPLATE/contributing.md

```diff
@@ -0,0 +1,171 @@
+# Deals for Devs Contribution Guide
+
```

Comment:
I accidentally said "Approved" after leaving feedback lol Can you update based on the comments below?